src/tools/recipe2plan.ts

```diff
@@ -22,13 +22,14 @@ import {assert} from '../platform/assert-node.js';
  */
 export async function recipe2plan(path: string): Promise<string> {
   return await Flags.withDefaultReferenceMode(async () => {
-    const manifest = await Runtime.parseFile(path);
+    const manifest = await Runtime.parse(`import '${path}'`);
```

Comment:
Why this change?

Reply:
`parseFile` doesn't seem to import other manifests.

Reply:
That's surprising. Do you know why? Do you think we can fix it?

Reply:
Testing things more -- you're right! This isn't the fix, I needed to call `Runtime.init(...)`.